test/onevariable.jl

```diff
@@ -138,14 +141,14 @@ Base.iszero(::SymbNumber) = false
     @test length.( TaylorSeries.fixorder(zt, Taylor1([1], 1)) ) == (2, 2)
     @test eltype(TaylorSeries.fixorder(zt,Taylor1([1]))[1]) == Taylor1{Int}
     @test TS.numtype(TaylorSeries.fixorder(zt,Taylor1([1]))[1]) == Int
-    @test TaylorSeries.findfirst(t) == 1
-    @test TaylorSeries.findfirst(t^2) == 2
-    @test TaylorSeries.findfirst(ot) == 0
-    @test TaylorSeries.findfirst(zt) == -1
-    @test TaylorSeries.findlast(t) == 1
-    @test TaylorSeries.findlast(t^2) == 2
-    @test TaylorSeries.findlast(ot) == 0
-    @test TaylorSeries.findlast(zt) == -1
+    @test findfirst(t) == 1
```

Comment:
Here too a (nested) `@testset` would be nice!

Reply:
Same comment as above... tests are overall messy...